### DISCUSSION TITLE
Impossible to load parser for help: Failed to load parser: uv_dlsym

### DISCUSSION THREAD

**Discussion by 3rfaan** [nvim-treesitter/nvim-treesitter#4598] [Q&A] [answered]
When I run the ":checkhealth" command inside Neovim I get this error message:

  - ERROR: Impossible to load parser for help: Failed to load parser: uv_dlsym: /home/arfan/.local/share/nvim/site/pack/packer/start/nvim-treesitter/parser/help.so: undefined symbol: tree_sitter_help

Does anyone have an idea what's causing this?

**Comment by amaanq** [comment, marked as answer]
It was renamed to vimdoc, please check the breaking changes pinned issue before posting issues/discussions: https://github.com/nvim-treesitter/nvim-treesitter/issues/2293

**Reply by 3rfaan** [reply]
Thanks, this solved my problem. Sorry for not noticing.

**Reply by benjamin051000** [reply]
Just for clarity, you can `:TSUninstall help` and `:TSInstall vimdoc` to remove the error in `:checkhealth` and get the new parser.
